CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+# [2.16.0] - 2023-11-09
+
+### Fixed
+- Android: fixed an issue that caused unnecessary recompositions in the Compose components.
+- Android: `EditorEditText` was being leaked by the ViewModel on activity recreations, also causing the right styles to not be applied properly.
+- Android: change several default theme colors.
+- Android: add `MentionsState` to get the intentional mentions in the text.
+- Android: make `LinkSpan`, `PillSpan` and `CustomMentionSpan` clickable in the TextView component.
+
+### Added
+- Added `MentionDetector` to the FFI bindings so we can use the same method for detecting mentions in all platforms.
+- Added `replaceSuggestion` and `insertMentionAtSuggestion` methods to the Compose implementation of the RTE composer.
```

Comment:
We usually prefix stuff that affects all platforms with `Common: `

Reply:
Fixed in https://github.com/matrix-org/matrix-rich-text-editor/pull/861/commits/3960ec69c40b2e32b09dcae6acd64bb36a4f31b5